nav2_map_server/include/nav2_map_server/map_saver.hpp

```diff
@@ -23,23 +23,53 @@
 namespace nav2_map_server
 {
 
+#define FREE_THRESH_DEFAULT 25
+#define OCCUPIED_THRESH_DEFAULT 65
+typedef struct
```

Comment:
Lets make this modern with `struct SaveParameters {};`

Reply:
Fixed in https://github.com/ros-planning/navigation2/commit/84cff7d8cf43443e23bc4852129791d665f6e5c7#diff-f278c22b4e54afc4d64216a9c1e5e802R74